ax/modelbridge/array.py

```diff
@@ -492,7 +492,12 @@ def _evaluate_acquisition_function(
         acq_options: Optional[Dict[str, Any]] = None,
     ) -> List[float]:
         return self._model_evaluate_acquisition_function(
-            X=observation_features_to_array(self.parameters, observation_features),
+            X=np.array(
+                [
+                    observation_features_to_array(self.parameters, obsf)
+                    for obsf in observation_features
+                ]
```

Comment:
This will throw an error if `observation_features` is `List[ObservationFeatures]`. Since the public method makes sure `observation_features` is `List[List[ObservationFeatures]]`, let's just change the signature to accept only that.